src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayRent = 40;
+  let sumForRent = 0;
+
+  if (days >= 7) {
+    sumForRent = oneDayRent * days - 50;
+
+    return sumForRent;
```

Comment:
The `return` statement makes the `else` unnecessary. Remove the `else` to simplify the code structure as per checklist item 5.

Reply:
but i don`t have "else" in code